packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn.tsx

```diff
@@ -11,4 +11,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'call-pstn',
```

Comment:
Can we rename this to something more generic that describes what the icon looks like 'call-user'?

Reply:
Since we're merging 2 icons into 1, can you also add an `exportedAs` to `icons-call-pstn-full` as well

Reply:
PSTN is public switched telephone network. It differs from user semantically, it is sub set of all users.

Reply:
And exportedAs already added to `icons-call-pstn-full`